soot-infoflow-summaries/summariesManual/java.util.concurrent.atomic.AtomicReferenceArray.xml

```diff
@@ -58,7 +58,12 @@
 			</flows>
 		</method>
 		<method id="void set(int,java.lang.Object)">
-			<flows>
+			<clears>
```

Comment:
Du setzt ein Element in einem Array. Deswegen kannst du nicht alle Taints auf dem Array wegwerfen.